docs/Introduction.md

```diff
@@ -1,21 +1,21 @@
 # Introduction
-**Subscription Management** - a turnkey solution that provides an easy and secure monetization mechanism for your app. Backed by Stripe, it provides an automated and integrated solution to manage your customers, subscriptions and handle payments.
+**Subscription Management** - is a turnkey solution that provides an easy and secure monetization mechanism for your Business Central app. Designed by Theta and backed by Stripe, it provides an automated and integrated solution to handle payments and manage your customers and subscriptions.
 
 ## Key Features 
-- **Backed by Stripe**, #1 payment platform available in all BC SaaS countires.
-- Offer **flexible billing models** including tiered, per seat and usage based pricing.
-- Supports **135+ currencies** and multiple payment intervals.
-- No middle tier required. Only **direct**, **secure**, end-to-end **integration** with app owner’s Stripe account.
+- **Backed by Stripe**, the #1 payment platform available in all Business Central SaaS countries.
+- Offers **flexible billing models** including tiered, per seat and usage based pricing. Any model you choose to provide your customers.
```

Comment:
"Any model you choose to provide your customers." This may confuse without instruction. Similar to "You select the currencies and payment intervals you accept.".

Reply:
I agree that may be true but in the introduction don't we want to be able to showcase all options?

Reply:
@MarkvanNoppen, I was looking for list of what available. As soon we get buy in on available feature, then there search to drill down into specifics.